src/community/edge_betweenness.c

```diff
@@ -226,9 +230,29 @@ igraph_error_t igraph_community_eb_get_merges(const igraph_t *graph,
                                    igraph_vector_int_t *membership) {
 
     igraph_integer_t no_of_nodes = igraph_vcount(graph);
+    igraph_integer_t no_of_edges = igraph_ecount(graph);
     igraph_vector_int_t ptr;
     igraph_integer_t i, midx = 0;
     igraph_integer_t no_comps;
+    igraph_integer_t no_removed_edges = igraph_vector_int_size(edges);
+    igraph_integer_t max_merges;
+    igraph_integer_t min;
+    igraph_integer_t max;
+
+    if (no_removed_edges > 0) {
+        igraph_vector_int_minmax(edges, &min, &max);
+        if (min < 0) {
+            IGRAPH_ERRORF("Edge ids should not be negative, found %" IGRAPH_PRId ".", IGRAPH_EINVAL, min);
```

Comment:
Let us not refer to negative edge IDs, as it does not make sense in high-level interfaces that use 1-based indexing. Can we say something like "out-of-bounds (too small)"? Any other suggestions?